Bash/InstallHuntress-macOS-bash.sh

```diff
@@ -136,39 +141,44 @@ shift $((OPTIND-1)) # remove parsed options and args from $@ list
 logger "=========== INSTALL START AT $dd ==============="
 logger "=========== $rmm Deployment Script | Version: $version ==============="
 
-# VALIDATE OPTIONS PASSED TO SCRIPT
+# validate options passed to script, remove all invalid characters except spaces are converted to dash
 if [ -z "$organization_key" ]; then
-    organizationKey=$(echo "$defaultOrgKey" | xargs)
-    logger "--organization_key parameter not present, using defaultOrgKey instead: $defaultOrgKey"
+    organizationKey=$(echo "$defaultOrgKey" | tr -dc '[:alnum:]- ' | tr ' ' '-')
+    organizationKey=$(echo "$organizationKey" | xargs)
```

Comment:
```suggestion
    organizationKey=$(echo "$defaultOrgKey" | tr -dc '[:alnum:]- ' | tr ' ' '-' | xargs)
```
i _think_ you should be able to combine these two.
(honestly with *sh scripts, I always have to test them out by hand)